### PR TITLE
Tell robots where to find the Help Centre sitemap

### PR DESCRIPTION
**Pull request by kelvin-chappell**
Ssh! This is secret until we set up the redirects.
So in draft state until then.

### CODE REVIEW

File: app/server/routes/core.ts [hunk 1]
```diff
@@ -33,7 +33,10 @@ router.get("/robots.txt", (_, res: Response) => {
     "User-agent: AdsBot-Google\n" +
     "User-agent: AdsBot-Google-Mobile\n" +
     "Disallow: /\n\n";
-  const prodAccessList = allowHelpCentre + disallowGoogleAdsBots;
+  const helpCentreSitemap =
+    "Sitemap: https://manage.theguardian.com/help-centre/sitemap.txt\n\n";
```
[the remark this answers is not in the document]
Comment (kelvin-chappell) [reply app/server/routes/core.ts:37]: @ripecosta 
In the [general guidelines section](https://developers.google.com/search/docs/advanced/sitemaps/build-sitemap#general-guidelines) of the sitemap documentation, it says:
> A sitemap can be posted anywhere on your site, but a sitemap affects only descendants of the parent directory.

[the remark this answers is not in the document]
Comment (kelvin-chappell) [reply app/server/routes/core.ts:37]: but it probably does make sense to put it at the site root so I'll move it.  I can see this being confusing in the future.

[the remark this answers is not in the document]
Comment (ripecosta) [reply app/server/routes/core.ts:37]: I knew you could have a sitemap for any children but thought you needed to least have a 'central' one in the root and you could then link to other ones further down tree from the central one.
But maybe you were going to point directly to sitemap location using the google console so the 'central' one isn't necessary?

[the remark this answers is not in the document]
Comment (kelvin-chappell) [reply app/server/routes/core.ts]: You can call a sitemap anything you like so I think there's no default way to find it.
You have to do one of these things to to register a sitemap with Google:
* add it to robots.txt
* ping Google's sitemap notifier URL
* manually register it in the Google console as you say
The first of these is the only way for non-Google bots but I'm not sure if anyone else actually uses sitemaps.

But having said that, I have put it at the site root now anyway.

[the remark this answers is not in the document]
Comment (ripecosta) [reply app/server/routes/core.ts]: That makes sense. Thanks for clarifying.

**Review by ripecosta — APPROVED** [review]
👍  Approving now so it's ready to go

**Pull request merged**